concepts/05 Widgets/Scheduler/030 Appointments/015 Appointment Types/020 All-Day Appointments.md

```diff
@@ -1,25 +1,27 @@
-An all-day appointment is a non-recurring appointment that covers the whole [timetable's period](/concepts/05%20Widgets/Scheduler/050%20Timetable.md '/Documentation/Guide/Widgets/Scheduler/Timetable/').
+An all-day appointment is an appointment that occurs during the whole day.
 
 ![Scheduler All-Day Appointment](/images/UiWidgets/Scheduler_AllDayAppointment.png)
 
-An appointment can also be marked as all-day by assigning **true** to the defining object's [allDay](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/allDay.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay') field. The [startDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/startDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate') field should also be present in this object; the [endDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/endDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate') is optional.
+To create such an appointment, set its [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) field to **true** and specify the start day in the [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate) field; the time value is optional: 
```

Comment:
Are we talking about appointment's data object? We should reflect this when talking about fields.

Could you please also explain, what does the following sentence mean: 'the time value is optional'?

Reply:
The time value of startDate and endDate are optional. Have corrected this paragraph.